docs/quick_start.md

```diff
@@ -45,48 +45,133 @@ $ docker run -it --rm --gpus all \
-2021-03-17 01:14:03.477 INFO[driver.py:236] init
-2021-03-17 01:14:06.375 INFO[server_local.py:81] Triton Server stopped.
-2021-03-17 01:14:06.376 INFO[entrypoint.py:327] Starting perf_analyzer...
-2021-03-17 01:14:06.376 INFO[analyzer.py:83] Profiling server only metrics...
-2021-03-17 01:14:06.388 INFO[server_local.py:64] Triton Server started.
+
+2021-05-11 02:02:58.194 INFO[entrypoint.py:98] Starting a local Triton Server...
+2021-05-11 02:02:58.203 INFO[server_local.py:64] Triton Server started.
+2021-05-11 02:03:02.609 INFO[server_local.py:81] Triton Server stopped.
+2021-05-11 02:03:02.610 INFO[analyzer_state_manager.py:118] No checkpoint file found, starting a fresh run.
+2021-05-11 02:03:02.610 INFO[analyzer.py:82] Profiling server only metrics...
+2021-05-11 02:03:02.618 INFO[server_local.py:64] Triton Server started.
+2021-05-11 02:03:05.766 INFO[gpu_monitor.py:72] Using GPU(s) with UUID(s) = { GPU-e35ba3d2-6eef-2bb9-e35c-6ef6eada4f11 } for profiling.
+2021-05-11 02:03:08.720 INFO[server_local.py:81] Triton Server stopped.
+2021-05-11 02:03:08.721 INFO[run_search.py:146] Will sweep both the concurrency and model config parameters...
+2021-05-11 02:03:08.721 INFO[run_search.py:289] Concurrency set to 1. Instance count set to 1, and dynamic batching is disabled.
+2021-05-11 02:03:08.736 INFO[server_local.py:64] Triton Server started.
+2021-05-11 02:03:14.512 INFO[client.py:80] Model add_sub_i0 loaded.
+2021-05-11 02:03:14.514 INFO[model_manager.py:211] Profiling model add_sub_i0...
+2021-05-11 02:03:16.578 INFO[gpu_monitor.py:72] Using GPU(s) with UUID(s) = { GPU-e35ba3d2-6eef-2bb9-e35c-6ef6eada4f11 } for profiling.
 .
 .
 .
 ```
 
-This will perform a search across various config parameters on the `add_sub` model. This takes over 40 minutes even on a TITAN RTX as Model Analyzer will try to find the search bounds automatically. When finished, it will generate the directory `./analysis_results` containing measured data and a summary report. The directory should be structured as follows. 
+This will perform a search across various config parameters on the `add_sub` model. This takes over 40 minutes even on a TITAN RTX as Model Analyzer will try to find the search bounds automatically. When finished, Model analyzer stores all of the profiling measurements it has taken in a binary file in a directory called `./checkpoints` by default.
+
+```
+$ ls -l checkpoints
+total 12
+-rw-r--r-- 1 root root 11356 May 11 13:00 0.ckpt
+```
+
+Refer to the [Checkpoints](./checkpoints.md) documentation for more details on how checkpoint files work. 
+
+## Generate tables and summary reports using `analyze`
+In order to generate tables and summary reports, use the `analyze` subcommand as follows.
+
+```
+$ mkdir analysis results
+$ model-analyzer analyze --analysis-models add_sub -e analysis_results
+```
+
+This will fill the directory `./analysis_results` with measured data and a summary report. The directory should be structured as follows. 
```

Comment:
"The measured data and summary report will be placed inside the `analysis_results` folder."